lib/helpers.js

```diff
@@ -77,6 +78,9 @@ export function extractColor(color?: ColorType) {
   if (color == null) {
     return null;
   }
+  if (Platform.OS === 'ios') {
```

Comment:
This still will break the current API. We use `Color` class from `art` (https://github.com/sebmarkbage/art/blob/master/core/color.js) to convert various strings with different color formats to their `rgba` representation.

Reply:
To fix this, you can use `toHEX` method and send strings with colors to the native side, on both: `iOS` and `Android`.

Reply:
Done with the changes. Kindly review